docs/UserGuide.md

```diff
@@ -477,6 +477,16 @@ You can change the default colour of the tag with `t/TAG:#HEX_COLOUR`
 Example: `t/friends:#FF0000` for a red colour tag
 </div>
 
+### `a/ALIAS`
+`ALIAS` can take any values except keywords that are reserved for command names.
+
+Example: `a/findAlex`
```

Comment:
this is not the full alias command

Reply:
Oh this one is under the `Parameter Constraints` section, cos during pe-d someone suggested to add alias and command to the `Parameter Constraints`